migrations/20161031223310_post_video_table.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class PostVideoTable extends AbstractMigration
+{
+    public function change()
+    {
+        $this->table('post_video')
+             ->addColumn('post_id', 'integer')
+             ->addColumn('form_attribute_id', 'integer')
+             ->addColumn('value', 'string', ['null' => true])
```

Comment:
I think this should be able to just use `post_text` .. see my comment on mappings above